ui/lib/core/addon/components/form-field.hbs

```diff
@@ -162,6 +162,40 @@
               <F.Error data-test-validation-error={{this.valuePath}}>{{this.validationError}}</F.Error>
             {{/if}}
           </Hds::Form::TextInput::Field>
+        {{else if (eq @attr.options.editType "textarea")}}
+          <Hds::Form::Textarea::Field
+            name={{@attr.name}}
+            @id={{@attr.name}}
+            @hasVisibilityToggle={{false}}
+            @value={{or (get @model this.valuePath) @attr.options.defaultValue}}
+            @isInvalid={{this.validationError}}
+            placeholder={{@attr.options.placeholder}}
+            {{on "input" this.onChangeWithEvent}}
+            data-test-input={{@attr.name}}
+            as |F|
+          >
+            {{#unless this.hideLabel}}
+              {{#if this.labelString}}
```

Comment:
@dchyun remember to remove the `this.hideLabel` and `this.labelString` guards (not needed anymore, after https://github.com/hashicorp/vault/pull/30788)